site2/docs/cookbooks-bookkeepermetadata.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
@eolivelli you need to add the page to the sidebar. otherwise it won't show up at the sidebar.

Reply:
Sidebar is at `site2/website/sidebars.json`